BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.1.0-rc1-03006-01
+2.1.0-rc1-03127-02
```

Comment:
We should move to 2.2 to match the Core repos in case there are branch-specific fixes in future prodcon builds.